spec/factories/response.rb

```diff
@@ -4,5 +4,56 @@
     training_module { 'alpha' }
     question_name { '1-1-4' }
     answers { [(1..5).to_a.sample] }
+
+    trait :correct do
+      correct { true }
+    end
+
+    trait :incorrect do
+      correct { false }
+    end
+
+    trait :confidence_check do
+      schema do
+        [
+          '9-9-9',
+          'confidence_check',
```

Comment:
Was an aim of this PR to have the database mirror the CMS better - using `page_type` instead of `assessments_type`? Or is the intention to do this in a separate PR?

Reply:
There is `Training::Question#assessments_type` called `confidence_check` - to be entirely accurate it should be `confidence_questionnaire` but, as you note, does not impact the test which keys off the new `question_type`.  Perhaps when we clean up AST and use of this struct we can clean it up then.

Reply:
> Was an aim of this PR to have the database mirror the CMS better - using `page_type` instead of `assessments_type`? Or is the intention to do this in a separate PR?

The intention of the PR is to migrate user answers to response table.  The other work is incidental.